src/api/decorators.py

```diff
@@ -20,12 +60,12 @@ def handle_feedback(request, region_slug, language_code):
                 {"error": "Either comment or rating is required."}, status=400
             )
         if rating == "up":
-            emotion = "Pos"
+            emotion = "POS"
```

Comment:
```suggestion
            emotion = feedback_emotions.POS
```
I think hardcoding the values doesn't make sense when we also have the constants. Import them with:
```
from cms.constants import feedback_emotions

Reply:
Good catch!